kusto/ingest/internal/resources/resources.go

```diff
@@ -174,11 +174,18 @@ func (m *Manager) Close() {
 }
 
 func (m *Manager) renewResources() {
-	tick := time.NewTicker(1 * time.Hour)
+	tickDuration := 30 * time.Second
+	tickTotal := 1 * time.Hour
+	tick := time.NewTicker(tickDuration)
+	count := 0 * time.Second
 	for {
 		select {
 		case <-tick.C:
-			m.fetchRetry(context.Background())
+			count += tickDuration
+			if count >= tickTotal {
```

Comment:
Making sure there's some code out there which - when trying to get the resources - tries to refresh it synchronously if the expected duration has expired by a factor of 2 or 3

Reply:
Another PR